src/Miso/Html/Element.hs

```diff
@@ -10,119 +10,138 @@
 -- Portability :  non-portable
 ----------------------------------------------------------------------------
 module Miso.Html.Element
-  ( -- *** Smart constructors
+  ( -- ** Smart constructors
       nodeHtml
     , nodeHtmlKeyed
-    -- *** Combinators
-    , doctype_
+    -- ** Document metadata
```

Comment:
I grouped the elements in the export list using the same categories that MDN reference uses